packages/calcite-components/src/components/tabs/tabs.scss

```diff
@@ -3,39 +3,20 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-tab-accent-color-hover: Specifies the component's accent color when hovered.
```

Comment:
None of these are used in this component so I don't think these parts should be included in the doc.